Mirror/Runtime/NetworkClient.cs

```diff
@@ -18,6 +18,14 @@ public class NetworkClient
 
         public static bool pauseMessageHandling;
 
+        // how often are we sending ping messages
+        // used to calculate network time and RTT
+        public float pingFrequency = 2.0f;
+        // average out the last few results from Ping
+        public int pingWindowSize = 10;
+
+        double lastPing = 0;
```

Comment:
maybe rename to lastPingTime so it's more obvious what this does? someone could confuse it for the last ping result, like 20ms